<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>97</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>97</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Quit">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>360</y>
      <width>97</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Devices</string>
    </property>
    <widget class="QPushButton" name="pushButton_Mode">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_File">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>114</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
    <widget class="QComboBox" name="InputDeviceCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="OutputDeviceCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Mode">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>154</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Ouput">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>74</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Ouput</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Input">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>34</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
    <widget class="QComboBox" name="FileCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>205</y>
      <width>481</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Channels</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>461</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
    <widget class="QPushButton" name="pushButton_ChannelAll">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>97</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ChannelFirst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>First</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ChannelNone">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>30</y>
       <width>97</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Panic">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>97</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Panic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>345</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>345</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
